### ISSUE TITLE
Outdated documentation for loggers

### ISSUE THREAD

**Issue by quinor** [PyTorchLightning/pytorch-lightning#1394]
Custom logger example doesn't work:
https://pytorch-lightning.readthedocs.io/en/latest/loggers.html#custom-logger

General loggers example doesn't work either due to missing parameters for the loggers.* class constructors:
https://pytorch-lightning.readthedocs.io/en/latest/pytorch_lightning.loggers.html#module-pytorch_lightning.loggers

**Comment by Borda** [comment]
@quinor sorry for the delay, mind sending PR with the update? :rabbit:

**Comment by rbracco** [comment]
I've had a lot of issues with outdated documentation. For instance with `lr_find` the docs say 2 different things in 2 different places and theyre both wrong. Maybe we could start an effort to overhaul the docs to reflect the latest changes.

The instructions in the trainer section say
```
# default used by the Trainer (no learning rate finder)
trainer = Trainer(auto_lr_find=False)

# call tune to find the lr
trainer.tune(model)
```

in the lr_finder section of Common Use Cases they say
```
model = MyModelClass(hparams)
trainer = Trainer()

# Run learning rate finder
lr_finder = trainer.tuner.lr_find(model)

# Results can be found in
lr_finder.results

# Plot with
fig = lr_finder.plot(suggest=True)
fig.show()
```

but neither work. It turns out lr_find is currently broken because it doesnt accept a datamodule and theres a fix in progress #3425
